doc/api/deprecations.md

```diff
@@ -831,6 +840,7 @@ is not included in this list will be considered invalid in compliance with
 [`console.error()`]: console.html#console_console_error_data_args
 [`console.log()`]: console.html#console_console_log_data_args
 [`crypto.createCredentials()`]: crypto.html#crypto_crypto_createcredentials_details
+[`crypto.DEFAULT_ENCODING`]: crypto.html#crypto_crypto_defaultencoding
```

Comment:
`#crypto_crypto_defaultencoding` -> `#crypto_crypto_default_encoding`?

Reply:
@jasnell This wasn't fixed in the nit-fix commit ;-)

https://nodejs.org/api/crypto.html#crypto_crypto_default_encoding is the actual link, so it indeed should be `#crypto_crypto_default_encoding` .